specs-go/config.go

```diff
@@ -10,6 +10,7 @@ type Spec struct {
 	Version string `json:"cdiVersion"`
 	Kind    string `json:"kind"`
```

Comment:
Do we want to call out the allowed character change for the class in a docstring?